### PR TITLE
improve implementation part

### PR DESCRIPTION
**Pull request by PaulBredl**
work on implementation
I also ran half of it through grammarly

### CODE REVIEW

File: paper/appendix.tex [hunk 1]
```diff
@@ -30,48 +30,48 @@ \section{Design}\label{sec:appendix-design}  % Felix
...
-            \item \textit{Schema Order}: Orders elements according to the order in the schema.
-            \item \textit{Priority Order}: Orders elements based on their priority (e.g. required properties first, deprecated properties last).
-            \item \textit{Data Order }: Orders elements the same way as it is in the data.
-        \end{itemize} \\
-        GUI Editor Maximum Depth & Determines the maximum level of nesting or hierarchy displayed in the GUI editor. \\
-        Debugging Active & For debugging purposes, users can enable or disable the debugging panel as needed.  \\
-        \bottomrule
-    \end{tabular}
-\end{table*}
+%\section{Implementation}\label{sec:appendix-implementation}
```
Comment (PaulBredl) [inline paper/appendix.tex:33]: I personally would remove those tables and link to our wiki

**Comment by beyond2void** [comment]
Other parts look good!

**Comment by PaulBredl** [comment]
@beyond2void could you approve if you think it looks good? :)

**Review by beyond2void — APPROVED** [review]

File: paper/implementation.tex [hunk 2]
```diff
@@ -1,88 +1,104 @@
...
 \subsection{Code Panel}\label{subsec:code-editor}
 
 The code editor is a GUI panel designed for editing the \cfgfiles.
 For this project, we use the \textit{Ace Editor}\cite{Ace-Editor} library to embed an interactive code editor into our user interface.
-% It is a browser based editor that matches and extends the features, usability and performance of existing native editors such as TextMate, Vim or Eclipse.
-% It can be easily embedded in any web page or JavaScript application.
-
-\subsubsection{Features}
-
+It provides useful features for our approach, such as syntax highlighting and code folding.
```
[the remark this answers is not in the document]
Comment (PaulBredl) [reply paper/implementation.tex:21]: We have :D

[the remark this answers is not in the document]
Comment (PaulBredl) [reply paper/implementation.tex:21]: image
a lot of the code is collapsed

File: paper/implementation.tex [hunk 3]
```diff
@@ -93,101 +109,110 @@ \subsubsection{Features}\label{subsubsec:gui-editor-features}
...
     \label{fig:schema_violation}
 \end{figure}
 
 \subsection{Schema preprocessing}\label{subsec:schema-preprocessing}
 
-To represent the schema in the GUI editor, we preprocess the schema.
+To represent the schema in the GUI editor, it is necessary to preprocess the schema as for some JSON schema keywords
+it is not immediately clear how to represent them in a GUI\@.
+For example, the \texttt{type} keyword can have multiple values, which represents a type union.
+Here there is no obvious GUI component that can represent this.
```
[the remark this answers is not in the document]
Comment (PaulBredl) [reply paper/implementation.tex:177]:
```suggestion
There is no obvious GUI component that represents this case.
```

**Review by beyond2void — APPROVED** [review]

**Pull request merged**